modules/widgets/contact-info.php

```diff
@@ -123,6 +124,10 @@ function widget( $args, $instance ) {
 				}
 			}
 
+			if ( '' != $instance['email'] ) {
```

Comment:
If we want to accept additional text or multiple email addresses, maybe we should make that a new field then, as well as modify the new Email address field to specify that one can use multiple email addresses.

That said, I'm not sure it will benefit many people. One single email address seems like a good standard to me.